src/utils/protobuf.util.ts

```diff
@@ -0,0 +1,33 @@
+import Long from 'long';
+
+const POSTGRES_MAX_BIGINT = BigInt('9223372036854775807');
+const POSTGRES_MIN_BIGINT = BigInt('-9223372036854775808');
+
+export class ProtobufUtil {
+  /**
+   * BigInt to Long, Long is protobuf int64
+   *
+   * NOTE: Should only be used for db ids
+   *
+   * @param id BigInt - db bigint
+   * @returns Long - protobuf int64
+   */
+  public static getInt64FromBigInt(id: bigint) {
```

Comment:
change you change this to support params as number?

like `id: bigint | number

Reply:
@Ko1103 This function only support bigint, since I want to it was used only for `database id: bigint`.

imo: 

1. Never use `Long (= protobuf int64)` for JS number.
2. Use int32 for JS number(integer).
3. Use string for JS bigint.

Reply:
@ThaddeusJiang understood, LGTM